src/src/operationWithArrays/hillel/FindOccurance.java

```diff
@@ -0,0 +1,31 @@
+package operationWithArrays.hillel;
+
+import java.util.*;
+
+public class FindOccurance {
+    public static void main(String[] args) {
+
+        List<String> wordList = Arrays.asList("bird", "cat", "dog", "bird", "fox", "cat", "cat", "dog", "bird");
+        List<Map<String, Object>> occurrenceList = findOccurrence(wordList);
+        System.out.println("Word occurrence in the list:");
+        for (Map<String, Object> occurrence : occurrenceList) {
+            System.out.println(occurrence.get("name") + ": " + occurrence.get("occurrence"));
+        }
+    }
+
+
+    public static List<Map<String, Object>> findOccurrence(List<String> wordList) {
+        Map<String, Integer> occurrenceMap = new HashMap<>();
+        for (String word : wordList) {
+            occurrenceMap.put(word, occurrenceMap.getOrDefault(word, 0) + 1);
+        }
+        List<Map<String, Object>> occurrenceList = new ArrayList<>();
```

Comment:
You do not need such a structure. To comply with the requirement `{name: "bird", occurrence: 2},` you can create a wrapper that holds 2 fields, like name and occurrence, and put them in a list or use only a map instead. If you choose to use a map, then do not put `name` and `occurrence` as the keys, just put the value and increment it each time.